README.md

```diff
@@ -23,7 +23,12 @@ inventory](https://docs.google.com/spreadsheets/d/118O7JeVEPaoVsCIxoWLdDTctcTCe4
 
 ## Adding a new core
 
-This repository updates, but does not create, collections. To add a new collection, create its config here and deploy to get the config up to the server. Then use the UI to create the collection. Finally, you can add the collection to the deploy scripts so that it will be updated in future deployments (https://github.com/pulibrary/pul_solr/blob/main/config/collections.yml).
+This repository updates, but does not create, collections. To add a new collection:
+- create the collection config here
+- deploy to get the config up to the server
+- use the UI to create the collection
+- add the collection to the deploy scripts so that it will be updated in future deployments (https://github.com/pulibrary/pul_solr/blob/main/config/collections.yml)
+- add the collection to the `config_map` in the relevant `/config/deploy/<env>.rb` file so that it will be backed up
```

Comment:
the config map is used to upload config sets from the deploy location into zookeeper, this is the only line of code that references it: https://github.com/pulibrary/pul_solr/blob/dfdeace1ea3b038faf81b68ee706805474bf9124/config/deploy.rb#L65

Reply:
backups use the collections configured in `deploy/env.rb` files, see

https://github.com/pulibrary/pul_solr/blob/0211f9868c72136aa741a63572d7dbb59926cc3c/lib/tasks/pul_solr.rake#L22
and 
https://github.com/pulibrary/pul_solr/blob/0211f9868c72136aa741a63572d7dbb59926cc3c/lib/pul_solr.rb#L6